vernac/vernacentries.ml

```diff
@@ -66,6 +66,8 @@ module DefAttributes = struct
     let (((((locality, deprecated), polymorphic), program), canonical_instance), typing_flags), using =
       parse Notations.(locality ++ deprecation ++ polymorphic ++ program ++ canonical_instance ++ typing_flags ++ using) f
     in
+    if Option.has_some deprecated then
+      Attributes.unsupported_attributes [CAst.make ("deprecated (use a notation)",VernacFlagEmpty)];
```

Comment:
```suggestion
      Attributes.unsupported_attributes [CAst.make ("deprecated (use a notation instead)",VernacFlagEmpty)];
```

Reply:
I'm not sure, I would understand "instead" as "instead of the deprecated attribute", whereas we still need a deprecated attribute, just on the notation rather than on the definition. No strong opinion though, so if you're confident it's better, I'll go for it.

Reply:
What about
```suggestion
      Attributes.unsupported_attributes [CAst.make ("deprecated (use a notation and deprecate that instead)",VernacFlagEmpty)];
```